src/support/SupportDialog.ts

```diff
@@ -87,7 +87,7 @@ export function showSupportDialog() {
 						])
 					}),
 				),
-				searchExecuted
+				searchExecuted && withContactOption
```

Comment:
maybe something for later but we could show it but gate it behind the upgrade dialog, otherwise you won't even know it's there

Reply:
yes, I was considering replacing it with a "want actual human support, then upgrade" prompt, but that's something for someone elses wish list

Reply:
I mean we have an upgrade dialog, just show it when clicking on "support" button would be enough?